test/spec/common/util/validationUtil.spec.js

```diff
@@ -0,0 +1,61 @@
+Bahmni.Registration.customValidator = {
+  "age.days": {},
+  "name": {},
+  "Telephone Number": {},
+  "date": {}
+};
+
+describe('ValidationUtil', function () {
+  var date = new Date();
+
+  var ValidationUtil = Bahmni.Common.Util.ValidationUtil;
+  var customValidator = Bahmni.Registration.customValidator;
+  var complexObject, objectConfiguration;
+  beforeEach(function () {
+    complexObject = {
+      "name": "valerio",
```

Comment:
You can also use [chance.js](http://chancejs.com/) or [faker.js](https://github.com/marak/Faker.js/) to generate random names and other data. 🎲